src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -332,6 +334,28 @@ private static void MutationFunctionsTestSetup(RecalcEngine engine, bool numberI
             engine.UpdateVariable("t_bs2", FormulaValue.NewTable(t_bsType.ToRecord()));
         }
 
+        private static void LoadDataTestSetup(RecalcEngine engine, bool numberIsFloat)

```

Comment:
>LoadDataTestSetup [](http://example.com/codeflow?start=28&length=17)

@MikeStall @carlosfigueira Any better suggestions on how to test load a table with data to run `Summarize` tests? I'm trying to keep `Summarize` tests as sharable as possible with PA. Without this loading stage, each test case in txt will have 20+ lines just to initialize a TableValue.

Reply:
what about mutation scripts (like what we use for patch/collect)?  Those have shared state.

Reply:
Mutation scripts are not run in Power Apps (issue with the JS test runner). At this point I'd rather have the tests with many lines in Summarize.txt instead of having this custom setup - that has the advantage of making the tests self-contained.